src/wtc-modal-view.js

```diff
@@ -125,7 +133,13 @@ class Modal {
    */
   open() {
     if (!this.state) {
-      this.modal.classList.add(this.optionalClass);
+      if (this.optionalClass) {
+        if (typeof this.optionalClass === "string")
+          this.modal.classList.add(this.optionalClass);
+        else if (typeof this.optionalClass === "object")
```

Comment:
Same as above:

`this.optionalClass instanceof Array`

Reply:
Done